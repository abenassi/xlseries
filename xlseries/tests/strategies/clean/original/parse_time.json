{
    "test_case3": [
        "'1986    1º trim. (1)",
        "'            2º trim. *",
        "'            3º trim. (2) *",
        "'            4º trim."
    ],

    "test_case4": [
        "2° Trim 07",
        "u' 3 Trim 07 2'",
        "4° Trim 07 ",
        "1° Trim 08 "
    ],

    "test_case5": [
        "1991    Ene.",
        "1991    Feb.",
        "1991    Mar. *",
        "1991    Abr.",
        "1991    May.",
        "1991    Jun.",
        "1991    Jul. (1)",
        "1991  - Ago.",
        "1991    Set.",
        "1991    Oct.",
        "1991    Nov.",
        "1991    Dic."
    ],

    "test_case5b": [
        "1991,01 ",
        "1991,02  ",
        "1991,03  ",
        "1991,04  ",
        "1991.05  ",
        "1991.06  ",
        "1991,07  ",
        "1991,08  ",
        "1991-09  ",
        "1991/10  ",
        "1991,11  ",
        "1991,12  "
    ],

    "test_case6": [
        "2004  ",
        "             I",
        "             II",
        "             III",
        "             IV"
    ],

    "test_case6b": [
        "2008    Año *",
        "Trimestre    I *",
        "             II * (1)",
        "             III (2) *",
        "             IV (3) * (4)"
    ],

    "test_case7": [
        "  1995   ",
        "(1) 1996",
        "1997 (2)",
        "(3) 1998 (4)",
        "   (5)  1999   (6)  * "
    ],

    "external_case1": [
        "1991    Ene.",
        "        Feb.",
        "        Mar.",
        "        Abr. *",
        "        May.",
        "        Jun. (1)",
        "        Jul.",
        "        Ago.",
        "        Set.",
        "        Oct.",
        "        Nov.",
        "        Dic."
    ],

    "external_case2": [
        "1991    Enero.",
        "        Febrero (1) * (4)",
        "        Marzo (2).",
        "        Abril",
        "        Mayo",
        "        Junio.",
        "        Julio.",
        "        Agosto (3).",
        "        Septiembre",
        "        Octubre * (1) *",
        "        Noviembre",
        "        Diciembre"
    ],

    "external_case3": [
        "2004             I  (1)",
        "2004             II  *",
        "2004             III * (2) *",
        "2004             IV"
    ],

    "external_case4": [
        "Ago-91",
        "Sep-91 *",
        "Oct-91 (2)",
        "Nov - 91",
        "1991 - Dec",
        "Ene - 1992",
        "Feb. - 1992"
    ],

    "external_case5": [
        "2003 I",
        "II",
        "III",
        "IV",
        "Año",
        "2004 I",
        "II"
    ],

    "external_case6": [
        "III 01",
        "IV 01",
        "I 02",
        "II 02"
    ],

    "external_case7": [
        "I 04",
        "II 04",
        "III 04",
        "IV 04",
        "I 05",
        "II 05",
        "III 05",
        "IV 05",
        "I 06",
        "II 06",
        "III 06",
        "IV 06"
    ],

    "external_case8": [
        "I 95",
        "II",
        "III",
        "IV",
        "I 96",
        "II",
        "III",
        "IV"
    ],

    "external_case9": [
        "I-04",
        "II-04",
        "III-04",
        "IV-04",
        "I-05",
        "*II-05",
        "III-05*",
        "IV-05",
        "I-06",
        "II.T.06",
        "III-06",
        "IV-06"
    ],

    "external_case10": [
        "2000/01 ",
        "2001/02",
        " 2002/03",
        "2003/04 "
    ],

    "external_case11": [
        "(2)I.S.03",
        "II.S.03*",
        "I.S.04",
        "*II.S.04",
        "I.S.05(1)"
    ],

    "external_case12": [
        "III.1963",
        "IV.1963",
        "I.1964",
        "II.1964",
        "III.1964"
    ]

}
