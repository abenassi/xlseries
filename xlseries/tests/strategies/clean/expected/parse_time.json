{
    "test_case3": [
        "arrow.get(1986, 1, 1)",
        "arrow.get(1986, 4, 1)",
        "arrow.get(1986, 7, 1)",
        "arrow.get(1986, 10, 1)"
    ],

    "test_case4": [
        "arrow.get(2007, 4, 1)",
        "arrow.get(2007, 7, 1)",
        "arrow.get(2007, 10, 1)",
        "arrow.get(2008, 1, 1)"
    ],

    "test_case5": [
        "arrow.get(1991, 1, 1)",
        "arrow.get(1991, 2, 1)",
        "arrow.get(1991, 3, 1)",
        "arrow.get(1991, 4, 1)",
        "arrow.get(1991, 5, 1)",
        "arrow.get(1991, 6, 1)",
        "arrow.get(1991, 7, 1)",
        "arrow.get(1991, 8, 1)",
        "arrow.get(1991, 9, 1)",
        "arrow.get(1991, 10, 1)",
        "arrow.get(1991, 11, 1)",
        "arrow.get(1991, 12, 1)"
    ],

    "test_case5b": [
        "arrow.get(1991, 1, 1)",
        "arrow.get(1991, 2, 1)",
        "arrow.get(1991, 3, 1)",
        "arrow.get(1991, 4, 1)",
        "arrow.get(1991, 5, 1)",
        "arrow.get(1991, 6, 1)",
        "arrow.get(1991, 7, 1)",
        "arrow.get(1991, 8, 1)",
        "arrow.get(1991, 9, 1)",
        "arrow.get(1991, 10, 1)",
        "arrow.get(1991, 11, 1)",
        "arrow.get(1991, 12, 1)"
    ],

    "test_case6": [
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 4, 1)",
        "arrow.get(2004, 7, 1)",
        "arrow.get(2004, 10, 1)"
    ],

    "test_case6b": [
        "arrow.get(2008, 1, 1)",
        "arrow.get(2008, 1, 1)",
        "arrow.get(2008, 4, 1)",
        "arrow.get(2008, 7, 1)",
        "arrow.get(2008, 10, 1)"
    ],

    "test_case7": [
        "arrow.get(1995, 1, 1)",
        "arrow.get(1996, 1, 1)",
        "arrow.get(1997, 1, 1)",
        "arrow.get(1998, 1, 1)",
        "arrow.get(1999, 1, 1)"
    ],

    "external_case1": [
        "arrow.get(1991, 1, 1)",
        "arrow.get(1991, 2, 1)",
        "arrow.get(1991, 3, 1)",
        "arrow.get(1991, 4, 1)",
        "arrow.get(1991, 5, 1)",
        "arrow.get(1991, 6, 1)",
        "arrow.get(1991, 7, 1)",
        "arrow.get(1991, 8, 1)",
        "arrow.get(1991, 9, 1)",
        "arrow.get(1991, 10, 1)",
        "arrow.get(1991, 11, 1)",
        "arrow.get(1991, 12, 1)"
    ],

    "external_case2": [
        "arrow.get(1991, 1, 1)",
        "arrow.get(1991, 2, 1)",
        "arrow.get(1991, 3, 1)",
        "arrow.get(1991, 4, 1)",
        "arrow.get(1991, 5, 1)",
        "arrow.get(1991, 6, 1)",
        "arrow.get(1991, 7, 1)",
        "arrow.get(1991, 8, 1)",
        "arrow.get(1991, 9, 1)",
        "arrow.get(1991, 10, 1)",
        "arrow.get(1991, 11, 1)",
        "arrow.get(1991, 12, 1)"
    ],

    "external_case3": [
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 4, 1)",
        "arrow.get(2004, 7, 1)",
        "arrow.get(2004, 10, 1)"
    ],

    "external_case4": [
        "arrow.get(1991, 8, 1)",
        "arrow.get(1991, 9, 1)",
        "arrow.get(1991, 10, 1)",
        "arrow.get(1991, 11, 1)",
        "arrow.get(1991, 12, 1)",
        "arrow.get(1992, 1, 1)",
        "arrow.get(1992, 2, 1)"
    ],

    "external_case5": [
        "arrow.get(2003, 1, 1)",
        "arrow.get(2003, 4, 1)",
        "arrow.get(2003, 7, 1)",
        "arrow.get(2003, 10, 1)",
        "arrow.get(2003, 1, 1)",
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 4, 1)"
    ],

    "external_case6": [
        "arrow.get(2001, 7, 1)",
        "arrow.get(2001, 10, 1)",
        "arrow.get(2002, 1, 1)",
        "arrow.get(2002, 4, 1)"
    ],

    "external_case7": [
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 4, 1)",
        "arrow.get(2004, 7, 1)",
        "arrow.get(2004, 10, 1)",
        "arrow.get(2005, 1, 1)",
        "arrow.get(2005, 4, 1)",
        "arrow.get(2005, 7, 1)",
        "arrow.get(2005, 10, 1)",
        "arrow.get(2006, 1, 1)",
        "arrow.get(2006, 4, 1)",
        "arrow.get(2006, 7, 1)",
        "arrow.get(2006, 10, 1)"
    ],

    "external_case8": [
        "arrow.get(1995, 1, 1)",
        "arrow.get(1995, 4, 1)",
        "arrow.get(1995, 7, 1)",
        "arrow.get(1995, 10, 1)",
        "arrow.get(1996, 1, 1)",
        "arrow.get(1996, 4, 1)",
        "arrow.get(1996, 7, 1)",
        "arrow.get(1996, 10, 1)"
    ],

    "external_case9": [
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 4, 1)",
        "arrow.get(2004, 7, 1)",
        "arrow.get(2004, 10, 1)",
        "arrow.get(2005, 1, 1)",
        "arrow.get(2005, 4, 1)",
        "arrow.get(2005, 7, 1)",
        "arrow.get(2005, 10, 1)",
        "arrow.get(2006, 1, 1)",
        "arrow.get(2006, 4, 1)",
        "arrow.get(2006, 7, 1)",
        "arrow.get(2006, 10, 1)"
    ],

    "external_case10": [
        "arrow.get(2000, 1, 1)",
        "arrow.get(2001, 1, 1)",
        "arrow.get(2002, 1, 1)",
        "arrow.get(2003, 1, 1)"
    ],

    "external_case11": [
        "arrow.get(2003, 1, 1)",
        "arrow.get(2003, 7, 1)",
        "arrow.get(2004, 1, 1)",
        "arrow.get(2004, 7, 1)",
        "arrow.get(2005, 1, 1)"
    ],

    "external_case12": [
        "arrow.get(1963, 7, 1)",
        "arrow.get(1963, 10, 1)",
        "arrow.get(1964, 1, 1)",
        "arrow.get(1964, 4, 1)",
        "arrow.get(1964, 7, 1)"
    ]

}
